module/condition/migrations/Version20201119103017.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Migration;
+
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20201119103017 extends AbstractErgonodeMigration
+{
+
+    public function up(Schema $schema): void
+    {
+        $languages = $this->getDictionaryActive();
+
+        $this->addSql(sprintf(
+            'WITH conditions_list as (
+                SELECT id, (\'{\'|| arr.position -1 || \',language}\')::text[] as path
+                FROM condition_set,
+                    jsonb_array_elements(conditions) with ordinality arr(item_object, position)
+                where arr.item_object::jsonb ->> \'type\' = \'PRODUCT_HAS_STATUS_CONDITION\')
+            UPDATE condition_set
+            set conditions = jsonb_set(conditions, conditions_list.path, \'%s\')
+            from conditions_list
+            where condition_set.id = conditions_list.id',
+            $languages
+        ));
+
+        $this->addSql(sprintf(
+            'WITH conditions_list as (
+                SELECT id, (\'{\'|| arr.position -1 || \',language}\')::text[] as path
+                FROM event_store,
+                     jsonb_array_elements(payload->\'conditions\') with ordinality arr(item_object, position)
+                where arr.item_object::jsonb ->> \'type\' = \'PRODUCT_HAS_STATUS_CONDITION\')
+            UPDATE event_store
+            set payload = jsonb_set(
+            payload, \'{conditions}\', jsonb_set(payload->\'conditions\', conditions_list.path, \'%s\')
+            )
+            from conditions_list
+            where event_store.id = conditions_list.id',
+            $languages
+        ));
+    }
+
+    /**
+     * @return array
+     */
+    public function getDictionaryActive(): string
+    {
+        $qb = $this->connection->createQueryBuilder()
```

Comment:
In fact, you don't need the connection service to execute the query. Just add the appropriate SQL to the above queries.